php/class-photonfill.php

```diff
@@ -779,17 +779,12 @@ public function get_breakpoint_url( $attachment_id, $img_size, $breakpoint, $pix
 
 		/**
 		 * Swap `lazyloaded` class for `lazyload`. Why is this necessary?
-		 * We need to unveil lazyload images in TinyMCE, which removes the `lazyload` class and adds the `lazyloaded` class. This means on the front end
-		 * lazysizes won't pick up on the image b/c it assumes the image has already been lazyloaded.
-		 * @param int $post_id
+		 * We need to unveil lazyload images in TinyMCE, which removes the `lazyload` class and adds the `lazyloaded` class.
+		 * This means on the front end lazysizes won't pick up on the image b/c it assumes the image has already been lazyloaded.
+		 * @param string $content
 		 */
-		public function swap_lazyload_classes( $post_id ) {
-			$current_post = get_post( $post_id );
-			$current_post->post_content = preg_replace( '/(class="[^"]*)(lazyloaded)([^"]*")/i', '$1' . 'lazyload' . '$3', $current_post->post_content );
-
-			remove_action( 'save_post', array( $this, 'swap_lazyload_classes' ) );
-			wp_update_post( $current_post );
-			add_action( 'save_post', array( $this, 'swap_lazyload_classes' ) );
+		public function swap_lazyload_classes( $content ) {
+			return preg_replace( '/(class=\\\\"[^"]*)(lazyloaded)([^"]*")/i', '$1' . 'lazyload' . '$3', $content );
```

Comment:
Wow, could they make it any more complicated??  Probably.  Thanks for the explanation.  Now I know!